src/components/Picklist/readme.md

```diff
@@ -317,3 +317,103 @@ initialState = { value: { name: 'option 3', label: 'Central Park' } };
     </GlobalHeader>
 </div>;
 ```
+
+##### Picklist with Modal
+
+```js
+import React from 'react';
+import { Picklist, Option, Modal, Input, Button, GoogleAddressLookup } from 'react-rainbow-components';
+import { Field, reduxForm } from 'redux-form';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBuilding } from '@fortawesome/free-solid-svg-icons';
+import styled from 'styled-components';
+
+initialState = { isOpen: false, selection: { name: '', label: '' } };
+
+const StyledInput = styled(Input)`
+    margin-bottom: 90px;
+    margin-top: 32px;
+`;
+
+const containerStyles = {
+    width: '220px'
+};
+
+function handlePicklistChange(value) {
+    if (value.name === 'option 1') {
+        setState({
+            isOpen: true,
+        });
+    } else {
+        setState({ selection: value });
+    }
+}
+
+<div className="rainbow-m-bottom_xx-large rainbow-p-bottom_xx-large">
+    <GlobalHeader
+        src="images/user/user3.jpg"
+        className="rainbow-p-bottom_xx-large rainbow-m-bottom_xx-large"
+        variant="neutral">
+        <div className="rainbow-flex rainbow-align_right">
+            <Picklist
+                id="picklist-9"
```

Comment:
In the examples the ids are put with odd ascending numbers, so that they are never repeated and also that structure is integrated with the integration tests.